src/foam/nanos/auth/DeletedAwareDAOTest.js

```diff
@@ -44,9 +44,7 @@ foam.CLASS({
       ],
       javaCode: `
         DAO delegate = new MDAO(DeletedAwareDummy.getOwnClassInfo());
-        DAO dao = (DAO) new DeletedAwareDAO.Builder(x)
-          .setDelegate(delegate)
-          .build();
+        DAO dao = (DAO) new DeletedAwareDAO(x, delegate);
```

Comment:
Here is an example where the js is preferred, as the builder is so helpful for configuration.